relayer-cli/src/components.rs

```diff
@@ -71,27 +75,29 @@ pub fn enable_ansi() -> bool {
 /// Builds a tracing filter based on the input `log_level`.
 /// Enables tracing exclusively for the relayer crates.
 /// Returns error if the filter failed to build.
-fn build_tracing_filter(log_level: String) -> Result<EnvFilter, FrameworkError> {
-    let target_crates = ["ibc_relayer", "ibc_relayer_cli"];
-
-    // SAFETY: unwrap() below works as long as `target_crates` is not empty.
-    let directive_raw = target_crates
-        .iter()
-        .map(|&c| format!("{}={}", c, log_level))
-        .reduce(|a, b| format!("{},{}", a, b))
-        .unwrap();
+fn build_tracing_filter(log_level: LogLevel) -> Result<EnvFilter, FrameworkError> {
```

Comment:
```suggestion
fn build_tracing_filter(default_level: LogLevel) -> Result<EnvFilter, FrameworkError> {
```
Might be useful to rename the argument for clarity, because it is ignored when the `RUST_LOG` environment variable is set.